### PR TITLE
Remove lines

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Review by nechaido — CHANGES_REQUESTED** [review]
Need some changes.

**Review by nechaido — CHANGES_REQUESTED** [review]
Don't like it.

**Review by nechaido — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -5,17 +5,3 @@
 ## Setup git repo
 
 ## Init npm, create package.json
-
-## Install Node.js
```
Comment (tshemsedinov) [inline README.md]: Ok, fixed @nechaido

Comment (nechaido) [reply README.md]: Nice, go ahead and merge.

**Pull request merged**